doc/schema.md

```diff
@@ -41,6 +41,10 @@ When the indexer is synced up to the tip of the chain, the hash of the tip is sa
 
  * `"t" →  "{blockhash}"`
 
+ Each MimbleWimble output results in following new row:
+
+ * `"MWO{output-id}" → "{serialized-output}"`
```

Comment:
Why use a 3 character prefix? All other row types use a single character.

Reply:
I thought it would be more descriptive (**M**imble**W**imble **O**utput)

Reply:
Not a place you need descriptiveness, though. It's wasteful to use more than a single character.